mir-it/src/test/java/org/mycore/mir/it/tests/MIRITBase.java

```diff
@@ -68,6 +68,7 @@ protected SolrClient getSolrClient() {
 
     protected static LukeResponse getLukeResponse(Core core) throws IOException, SolrServerException {
         LukeRequest request = new LukeRequest();
+        request.setBasicAuthCredentials("admin", "alleswirdgut");
```

Comment:
Use properties for values

Reply:
Its a test. Why use properties in tests?